src/main/java/io/nats/client/api/MessageGetRequest.java

```diff
@@ -79,13 +86,14 @@ public MessageGetRequest(long sequence) {
      */
     @Deprecated
     public MessageGetRequest(String lastBySubject) {
-        this(-1, lastBySubject, null);
+        this(-1, lastBySubject, null, null);
     }
 
-    private MessageGetRequest(long sequence, String lastBySubject, String nextBySubject) {
+    private MessageGetRequest(long sequence, String lastBySubject, String nextBySubject, ZonedDateTime startTime) {
```

Comment:
Probably should add a builder to this class, especially if we are going to add a generic get message api

Reply:
Added the builder :+1:

Reply:
And removed it :innocent: 
https://github.com/nats-io/nats.java/pull/1226/files#r1763803353